kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -2409,6 +2411,7 @@ protected void handleWriteTxnMarkers(KafkaHeaderAndRequest kafkaHeaderAndRequest
                     ctx);
             getReplicaManager().appendRecords(
                     kafkaConfig.getRequestTimeoutMs(),
+                    kafkaConfig.getRequiredAcks(),
```

Comment:
We don't need to add a server-side config for it. Just pass 1 because we should not skip the callback of `partitionLog#addPartitionResponse` here.

Reply:
sorry,  I'll submit the unit test later and  pass 1 here.